analytics.js

```diff
@@ -31,7 +31,7 @@ class Analytics {
     const useragent = this.store.get('user-agent');
 
     if (process.env.NODE_ENV !== 'development') {
-      if (global.getUserSettings.collectStatistics) {
+      if (this.store.get('userPrefs.app.analytics.collectStatistics')) {
```

Comment:
this should be good for now. Let's think of how to deprecate the old store completely.